ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/BlockDataGenerator.java

```diff
@@ -619,7 +620,7 @@ public KeyPair generateKeyPair() {
     private boolean hasTransactions = true;
     private TransactionType[] transactionTypes = TransactionType.values();
     private Optional<Address> coinbase = Optional.empty();
-    private Optional<Long> baseFee = Optional.empty();
+    private Optional<Optional<Long>> maybeBaseFee = Optional.empty();
```

Comment:
Yeah because we need a way to encode if the basefee is unspecified. This is important for knowing whether to use a default base fee or the optional.empty the caller gives explicitly to the builder.